### PR TITLE
showing images in wagtail admin

### PR DESCRIPTION
**Pull request by codiebeulaine**
@justinvdm ready for review

### CODE REVIEW

File: mentorship/mentorship/urls.py [hunk 1]
```diff
@@ -13,20 +13,46 @@
...
+    from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+
+    urlpatterns += staticfiles_urlpatterns()
+    urlpatterns += static(
+        settings.MEDIA_URL + 'images/', document_root=os.path.join(
+            settings.MEDIA_ROOT, 'images'))
+    urlpatterns += patterns(
+        '',
+        (r'^favicon\.ico$', RedirectView.as_view(
+            url=settings.STATIC_URL + 'myapp/images/favicon.ico'))
```
Comment (justinvdm) [inline mentorship/mentorship/urls.py]: Should `myapp` be here?

Comment (codiebeulaine) [inline mentorship/mentorship/urls.py]: sorry forgot to take that out

File: mentorship/mentorship/settings.py [hunk 2]
```diff
@@ -234,6 +234,11 @@
 
 WAGTAIL_SITE_NAME = ''
 
+WAGTAILIMAGES_FEATURE_DETECTION_ENABLED = False
+
+MEDIA_ROOT = join(PROJECT_ROOT, 'media')
+MEDIA_URL = '/media/'
```
Comment (justinvdm) [inline mentorship/mentorship/settings.py]: Will this not cause the current pictures store at `/images` in qa and prd to no longer be found (unless we manually move them)? Wondering if we should rather just use the project route as the media route as we do at the moment. Thoughts?

**Comment by codiebeulaine** [comment]
screen shot 2016-12-22 at 12 26 22 pm

**Comment by justinvdm** [comment]
Two minor comments, looks good :)

File: mentorship/mentorship/settings.py [hunk 3]
```diff
@@ -236,8 +236,8 @@
 
 WAGTAILIMAGES_FEATURE_DETECTION_ENABLED = False
 
-MEDIA_ROOT = join(PROJECT_ROOT, 'media')
-MEDIA_URL = '/media/'
+MEDIA_ROOT = join(PROJECT_ROOT)
```
Comment (justinvdm) [inline mentorship/mentorship/settings.py]: Could we remove `join()`? I don't think we need it anymore.

**Comment by justinvdm** [comment]
:+1: when travis is happy and that last minor comment of mine is resolved

**Pull request merged**